web/shapeworks/src/components/DatasetSearch.vue

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+import { defineComponent, ref } from '@vue/composition-api'
+import router from '@/router';
+import { getDatasets } from '@/api/rest';
+import { loadingState, allDatasets } from '@/store';
+
+
+export default defineComponent({
+    setup() {
+      const searchText = ref(router.currentRoute.params.searchText)
+
+      async function navigateToResults() {
+        router.replace('/search/'+searchText.value)
+        loadingState.value = true;
+        allDatasets.value = (await getDatasets(searchText.value)).sort((a, b) => {
+            if(a.created < b.created) return 1;
+            if(a.created > b.created) return -1;
+            return 0;
+        });
+        loadingState.value = false;
+      }
+
+      return {
+        searchText,
+        navigateToResults
+      }
+    }
+})
+</script>
+
+<template>
+    <v-chip>
+        <v-text-field
+            autofocus
+            rounded
+            hide-details
+            v-model="searchText"
+            @input="navigateToResults"
+        >
+            <template v-slot:prepend-inner>
+            <v-icon color="blue">
```

Comment:
The blue is pretty eye-catching and seems a little too prominent. This could be a more muted blue to match better.

Reply:
Changed in [aa9e131](https://github.com/girder/shapeworks-cloud/pull/311/commits/aa9e1310fca83505649ba82d49456a31f58bb46e)